include/psp2/paf.h

```diff
@@ -25,6 +37,11 @@ void *sce_paf_private_memmove(void *destination, const void *source, SceSize num
 void *sce_paf_private_bcopy(void *destination, const void *source, SceSize num);
 void *sce_paf_private_memset(void *ptr, int value, SceSize num);
 int sce_paf_private_snprintf(char *s, SceSize n, const char *format, ...);
+
+int sce_paf_private_vsnprintf(char *dst, unsigned int max, const char *fmt, va_list arg);
+
+#define sce_paf_vsnprintf sce_paf_private_vsnprintf
```

Comment:
do we need this macro?

Reply:
Yes, for bruteforce names.
I'm thinking of removing `private_` from all paf function names in the future.